sdk/rust/oak_tests/src/lib.rs

```diff
@@ -22,24 +22,34 @@ use oak_runtime::ChannelEither;
 use protobuf::{Message, ProtobufEnum};
 use std::collections::HashMap;
 use std::process::Command;
-use tempdir::TempDir;
+
+use serde::Deserialize;
 
 // TODO(#544): re-enable unit tests of SDK functionality
 
+#[derive(Deserialize)]
+struct CargoMetadata {
+    target_directory: String,
+}
+
 /// Uses cargo to compile a Rust manifest to Wasm bytes. Compilation is performed in a temporary
 /// directory.
 pub fn compile_rust_wasm(cargo_path: &str, module_name: &str) -> std::io::Result<Vec<u8>> {
-    let temp_dir = TempDir::new("")?;
+    let cargo_output = Command::new("cargo").arg("metadata").output()?;
```

Comment:
Could do with a comment explaining what's going on here.

Reply:
Switched to carg_metadata